.github/workflows/ci.yml

```diff
@@ -731,25 +728,28 @@ jobs:
         run: |
           conda info
           conda list
-      - run: pytest -n 2 modin/pandas/test/dataframe/test_binary.py
-      - run: pytest -n 2 modin/pandas/test/dataframe/test_default.py
-      - run: pytest -n 2 modin/pandas/test/dataframe/test_indexing.py
-      - run: pytest -n 2 modin/pandas/test/dataframe/test_iter.py
-      - run: pytest -n 2 modin/pandas/test/dataframe/test_join_sort.py
-      - run: pytest -n 2 modin/pandas/test/dataframe/test_map_metadata.py
-      - run: pytest -n 2 modin/pandas/test/dataframe/test_pickle.py
-      - run: pytest -n 2 modin/pandas/test/dataframe/test_reduce.py
-      - run: pytest -n 2 modin/pandas/test/dataframe/test_udf.py
-      - run: pytest -n 2 modin/pandas/test/dataframe/test_window.py
-      - run: pytest -n 2 modin/pandas/test/test_api.py
-      - run: pytest -n 2 modin/pandas/test/test_concat.py
-        if: matrix.engine == 'python'
-      - run: pytest modin/pandas/test/test_concat.py # Ray and Dask versions fails with -n 2
-        if: matrix.engine != 'python'
-      - run: pytest -n 2 modin/pandas/test/test_general.py
-      - run: pytest -n 2 modin/pandas/test/test_groupby.py
-      - run: pytest -n 2 modin/pandas/test/test_rolling.py
-      - run: pytest -n 2 modin/pandas/test/test_series.py
+      - name: Start local ray cluster
+        run: |
+          ray start --head --port=6379 --object-store-memory=1000000000
+          echo "MODIN_RAY_CLUSTER=True" >> $GITHUB_ENV
+        if: matrix.engine == 'ray'
+      - run: pytest modin/pandas/test/dataframe/test_binary.py
```

Comment:
isn't it going to run dask serially as well?

Reply:
Yeah I realize that now, I should change it